package-lock.json

```diff
@@ -9,7 +9,7 @@
       "version": "5.8.2",
       "license": "ISC",
       "dependencies": {
-        "@balena/design-tokens": "^0.7.2",
```

Comment:
This file contains numerous changes that are unrelated to the module installation. Could you please install only the specific module without running npm install?